cfg/cfg.go

```diff
@@ -38,10 +38,13 @@ var Config Configuration
 
```

Comment:
Regarding line 52: It seems you are right. I didn't catch that a config file can be given via command line option. My bad. I will update the PR.

Regarding line 64: This will only get executed, if the type assertion of err being a viper.ConfigFileNotFoundError fails. So if something else fails besides the config file just not being found. Note the !ok on line 62.

Regarding line 68: We get to that code block if on line 62 the err was a viper.ConfigFileNotFoundError.

I did only test avash reading the config file with .yml extension from the default locations ($HOME, /etc/avash). I didn't notice the exit based on the cli parameter. Again, my bad. Will update the PR.

Reply:
I think I addressed your concern about exiting when the cli parameter ends in .yml.
The rest works on my machine as I would expect it to.